gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -517,6 +517,66 @@ object ExpressionConverter extends SQLConfHelper with Logging {
           replaceWithExpressionTransformer(sha2.left, attributeSeq),
           replaceWithExpressionTransformer(sha2.right, attributeSeq),
           sha2)
+      case namedStruct: CreateNamedStruct =>
```

Comment:
Just move it? Not relevant change?

Reply:
The other expression case must be put before LeafExpression, UnaryExpression, BinaryExpression, TernaryExpression, QuaternaryExpression, for example, ElementAt is extends with BinaryExpression, if deal BinaryExpression first, there will be error.

Reply:
I think we should move `ElementAt` to `BinaryExpressionTransformer.scala`, not here, maybe in another PR